apps/main/src/components/radio-group.tsx

```diff
@@ -3,7 +3,6 @@ import { For } from "solid-js"
 function RadioGroup<V>(props: {
   class: string
   name: string
```

Comment:
Surprisingly, removing these extra attrs helped VoiceOver announce the legend's value of the fieldset